modelgauge/api_server.py

```diff
@@ -0,0 +1,104 @@
+import itertools
+import multiprocessing
+import os
+from typing import Sequence, Optional
+
+from fastapi import FastAPI, Depends, HTTPException
+from fastapi.security import APIKeyHeader
+from pydantic import BaseModel
+
+from modelgauge.annotator import CompletionAnnotator
+from modelgauge.annotator_registry import ANNOTATORS
+from modelgauge.config import load_secrets_from_config
+from modelgauge.load_plugins import load_plugins
+from modelgauge.prompt import TextPrompt
+from modelgauge.sut import PromptResponseSUT
+from modelgauge.sut_registry import SUTS
+from modelgauge.suts.together_client import CHAT_MODELS
```

Comment:
Do you envision this server only being used to interact with Together models? If so, we should probably move this file to the together plugin, because I don't think anything in the core of modelgauge should depend on an optional plugin.

Reply:
Interesting point. Let's discuss this in standup tomorrow. I agree this violates the initial intention of the breakdown for sure, but I'm not sure what to do about that.

Reply:
Could we just import the entire SUT registry, like how the file imports the ANNOTATORS registry?